rentals.json

```diff
@@ -0,0 +1 @@
+[{"date":"2020/09/09","book":0,"book_title":"kk","person":0,"person_name":"Eda"}]
```

Comment:
- [ ] (optional) Kindly consider fixing the rentals, so that creating a rental and retrieving it for display is made possible

Rental Created
![image](https://github.com/aima98/OOP-School-Library1/assets/81932478/2041cd30-8cb2-45e4-bde0-1e3ef955c9b3)

No Book Found On Retrieval
![image](https://github.com/aima98/OOP-School-Library1/assets/81932478/d0b982b8-7904-43eb-a841-1746f3c2a958)